modules/ROOT/pages/index.adoc

```diff
@@ -14,8 +14,9 @@ On-premise Mule applications cannot use Object Store v2.
 
 Object Store v2 features:
 
-* Persists keys for 30 days unless updated. If a key is updated, the TTL (time-to-live) is extended by another 30 days. For more information, see xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
-
+* Supports both static and rolling TTL (time-to-live) for data persistence.
```

Comment:
Our Term Presentation List shows this:

TTL (time to live, time-to-live value)

I looked it up while editing #51 and found mostly Time to Live and Time to live, but also time to live.
I don't care whether we use time to live or Time to Live, but we should not hyphenate it except as a compound adjective modifying a noun.

https://www.google.com/search?q=ttl&oq=ttl&aqs=chrome..69i57j0j46j0l3j69i61l2.1443j0j7&sourceid=chrome&ie=UTF-8

Reply:
```suggestion
* Supports both static and rolling TTL (time to live) for data persistence.
```